packages/sns/src/types/governance.params.ts

```diff
@@ -13,3 +13,10 @@ export interface SnsListNeuronsParams extends QueryParams {
   /** Index the search to returns a list that starts after specified neuron id */
   beforeNeuronId?: NeuronId;
 }
+
+/**
+ * The parameters to get an sns neuron
+ */
+export interface SnsGetNeuronParams extends QueryParams {
```

Comment:
you should export it in `index.ts` as well?

Reply:
this one and the SnsGovernanceError as well now that I think about it. Thanks!

Reply:
done